docs/en/maintain/upgrade.md

```diff
@@ -1,19 +1,20 @@
 # Upgrade
 
 Here is the impact when upgrading OpenMLDB:
-* If the created table is a single copy, it is not readable and writable during the upgrade process
-* If the created table is multi-replica, the read request on the upgraded node will fail briefly, and the write request will have a small amount of data loss.If short read failures are not tolerated, execute offlineendpoint before stopping each tablet node. If a small amount of write loss cannot be tolerated, write operations need to be stopped during the upgrade process
+* If the table is single-replica, we will add an extra replica before upgrading and delete it afterwards
```

Comment:
if someone can accept unavailable in short time, add extra replica is an unrequited operation as it may cause `OOM`

Reply:
In this case, we ask users to skip the `pre-upgrade` and `post-upgrade` steps?